library/cohesity_job.py

```diff
@@ -496,6 +496,39 @@ def get_view_storage_domain_id(module, self):
     except Exception as error:
         raise__cohesity_exception__handler(error, module)
 
+def update_indexing(module, payload):
+    """
+    """
+    payload['indexingPolicy'] = {
+        "disableIndexing": module.params.get('disable_indexing'),
+        "allowPrefixes": ["/"],
```

Comment:
These defaults are mentioned in the documentation right ?

Reply:
Will update functionality cause override or append to existing one. Either ways make the documentation clear.

Reply:
Will override existing values. I have updated the same in documentation as well.